### PR TITLE
🤖 Meca frontmatter

### PR DESCRIPTION
**Pull request by fwkoch**
This is the frontmatter part of the MECA export. It adds a new export format `meca` as well as `requirements` and `resources` to project frontmatter to list corresponding files to each of those categories.

### CODE REVIEW

**Comment by stevejpurves** [comment]
@fwkoch how does the `meca: export` relate to the project-wide export of a single meca bundle? 

i.e. if i want to build and deploy a site, and want a meca bundle to be built for it (i.e. the root article in the `_tcc.yml`) would be the entrypoint, where would I specify that I want the meca built?

is this a case of adding `exports.format: meca` to the root article's fromtmatter? and if so will everything in the toc be included?

**Comment by stevejpurves** [comment]
also `validateProjectFrontmatter` (https://github.com/executablebooks/mystjs/blob/92fd39dfee982edbeb51008cbbe37cda41cf09ab/packages/myst-frontmatter/src/frontmatter/validators.ts#L853) doesn't seem to be ever called in myst-cli, the only usage of the validation is when checking page frontmatter and via the `validateProjectFrontmatterKeys` functon. 

So it seems like there is no point currently where project frontmatter from `myst.yml` is validated?

File: packages/myst-frontmatter/src/frontmatter/validators.ts [hunk 1]
```diff
@@ -816,6 +818,8 @@ export function validateProjectFrontmatterKeys(
       validateThebe,
     );
   }
+
+  console.log('REQUIREMENTS', value);
```
Comment (stevejpurves) [inline packages/myst-frontmatter/src/frontmatter/validators.ts:822]: to remove ☝🏼 :(

**Comment by rowanc1** [comment]
@fwkoch there are some new changes in `main` that are likely needed for tests to pass -- SPDX released a new license list that we test against, and the jtex template download has been given a larger timeout.

**Comment by fwkoch** [comment]
Superseded by #431

**Pull request closed without merging**